lib/ai/actions/chat-actions.ts

```
@@ -19,39 +32,22 @@ export async function createChat({
+      finish_reason: finishReason,
+      sharing: 'private' as const,
+      updated_at: Date.now(),
+    };
 
-        if (updateError) {
-          console.error('Error updating chat:', updateError);
-        }
-        return;
-      }
+    const { success, error } = await convex.action(api.chats.createChatAction, {
+      serviceKey: process.env.CONVEX_SERVICE_ROLE_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)